landing-page/config.toml

```diff
@@ -22,7 +22,7 @@ sectionPagesMenu = "main"
 [[params.social]]
   title = "slack"
   icon = "slack"
-  url = "https://join.slack.com/t/apache-iceberg/shared_invite/zt-1jyaasx2a-TxE4z_ubxDkTFS7UFDHnjw"  
+  url = "https://join.slack.com/t/apache-iceberg/shared_invite/zt-1znkcg5zm-7_FE~pcox347XwZE3GNfPg"
```

Comment:
why this is not the latest link?

I can see that other places in this PR we are using zt-2561tq9qr-UtISlHgsdY3Virs3Z2_btQ